javascript/javascript1/week2/java.js

```diff
@@ -0,0 +1,146 @@
+//Flight booking fullname function
+
+function getFullname(firstname, surname, sex, useFormalName = false) {
+  let prefix = "";
+
+  if (useFormalName) {
+    if (sex === "male") prefix = "Mr";
+    else if (sex === "female") prefix = "Mrs";
+    else console.log("Please enter a valid gender");
+  }
+
+  return `${prefix} ${firstname} ${surname}`;
+}
+fullname1 = getFullname("Benjamin", "Hughes", "male");
+fullname2 = getFullname("Beatrice", "Jackson", "female", true);
+
+console.log(fullname1);
+console.log(fullname2);
```

Comment:
This works, but when `useFormalName` is `false`, then there is a space in front of `firstname` which is unnecessary